pkg/testing-helper/framework-helper/wrappers.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
nit, should we update the Copyright header to be godel?

Reply:
good catch, this should be Godel considering that it is our proprietary file.

Reply:
fixed, PTAL